react-frontend/src/components/Charts.tsx

```diff
@@ -125,21 +152,42 @@ function Charts({ a_n, b_n, limit, convergesTo, errorData, deltaData, toggleDisp
 		let cleanInput = input.indexOf('≈') >= 0 ? input.substring(0, input.indexOf('≈')) : input;
 		// replace root of if it wraps a sub expression in parens first since it's a more specific match
 		if (cleanInput.indexOf('root of (') > -1) {
-			cleanInput.replace(/root\sof\s\(([^)]+)\)(.*)/g, 'sqrt($1)$2');
+			cleanInput = cleanInput.replace(/root\sof\s\(([^)]+)\)(.*)/g, 'sqrt($1)$2');
 		}
 		// then go after the entire expression
 		if (cleanInput.indexOf('root of ') > -1) {
-			cleanInput.replace(/root\sof\s(.*)/g, 'sqrt($1)');
-		}
-		if (cleanInput.indexOf('_') > -1) {
-			cleanInput.replace(/_\(\w\)/, '[' + '$1' + ']');
+			cleanInput = cleanInput.replace(/root\sof\s(.*)/g, 'sqrt($1)');
 		}
+		cleanInput = wolframTextCleanup(cleanInput);
 		try {
 			const mathy = parse(cleanInput).toTex();
 			return `$$${mathy}$$`;
 		} catch (e) {
-			console.log(`failed to parse ${cleanInput}`);
+			console.error(`failed to parse ${cleanInput}`);
+		}
+	};
+
+	const wolframTextCleanup = (input: string) => {
```

Comment:
were having issues with underscores not properly converting the subsequent text to subscript, this fixes that bug

Reply:
Nit: Your comment here is a good comment to add to the code if you end up making other changes to the file.